src/azure-cli/azure/cli/command_modules/botservice/_params.py

```diff
@@ -166,6 +166,29 @@ def load_arguments(self, _):
         c.argument('enable_enhanced_auth', help='If true, enables enhanced authentication features. Must be true for --trusted-origins parameters to work.', arg_type=get_three_state_flag())
         c.argument('trusted_origins', nargs='+', help='Space separated Trusted Origins URLs (must use HTTPS) e.g. --trusted-origins https://mybotsite1.azurewebsites.net https://mybotsite2.azurewebsites.net')
 
+    with self.argument_context('bot directline site create') as c:
+        c.argument('enable_enhanced_auth', help='If true, enables enhanced authentication features. Must be true for --trusted-origins parameters to work.', arg_type=get_three_state_flag())
+        c.argument('is_enabled', options_list=['--is-enabled'], arg_type=get_three_state_flag(), help='Add the Direct Line Site in an enabled state.')
```

Comment:
```suggestion
        c.argument('is_enabled', options_list=['--enabled'], arg_type=get_three_state_flag(), help='Add the Direct Line Site in an enabled state.')
```
I guess `enabled` should be enough to represent your indent to user.

Reply:
Good catch! Done.